SIT.Manager.Avalonia/Services/LocalizationService.cs

```diff
@@ -74,6 +75,21 @@ public string TranslateSource(string key, params string[] replaces)
                     }
                 }
             }
+            else // if the translation was null in selected translation we will try to get that key in default English.
+            {
+                resourceInclude = CreateResourceLocalization("en-US");
+                if (resourceInclude.TryGetResource(key, null, out object? translationDefault))
+                {
+                    if (translationDefault != null)
+                    {
+                        result = (string)translationDefault;
+                        for (int i = 0; i < replaces.Length; i++)
+                        {
+                            result = result.Replace($"%{i + 1}", replaces[i]);
+                        }
+                    }
+                }
+            }
```

Comment:
Theres a lot of duplicate code here. Could we refactor this method to condense this down and avoid repeating ourselves?